src/emu/softlist.h

```diff
@@ -213,12 +212,12 @@ class software_list_device : public device_t
 
 	// getters that may trigger a parse
 	const char *description() { if (!m_parsed) parse(); return m_description.c_str(); }
-	bool valid() { if (!m_parsed) parse(); return m_infolist.count() > 0; }
+	bool valid() { if (!m_parsed) parse(); return m_infolist.size() > 0; }
```

Comment:
`std::list::size()` may be O(n) complexity, you should use `std::list::empty()` if you only care about whether it's zero or not